src/components/base/outline-alert/outline-alert.ts

```diff
@@ -0,0 +1,78 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import componentStyles from './outline-alert.css.lit';
+import { OutlineElement } from '../outline-element/outline-element';
+
+export const alertSizes = ['small', 'large'] as const;
```

Comment:
I think this is over complicating things. I think you want to replace these two lines with `export type AlertSize = 'small' | 'large'`

Reply:
Thanks for the review @LeeMellon. Going to look into all this in more detail when I get a chance.

The attempt here was to provide an array of values used by Storybook to populate the select box and provide the type in this file for TS checking.

Is there a way to do both of those in one place? This pattern I found somewhere, but was new to me. :) The best I could find to avoid copy & paste issues.

Reply:
No this is good, he's using [const assertion](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) here. I just figured out how to do this myself this week, it allows us to create a type and capture it in a variable in one go.

Reply:
Thanks for the explanation, @mabry1985! And the link to where it's used.

Reply:
Thanks both of you. Using this regularly now.